docs/resources/Channel.md

```diff
@@ -802,7 +802,7 @@ This endpoint supports requests with `Content-Type`s of both `application/json`
 ###### Parameters for `content-type: multipart/form-data`
 
 > info
-> Note that when sending `multipart/form-data`, you must provide a value for at **least one of** `content`, `embed` or `file`. For a `file` attachment, the `Content-Disposition` subpart header MUST contain a `filename` parameter.
+> Note that when sending `multipart/form-data`, you must provide a value for at **least one of** `content` or `file`. For a `file` attachment, the `Content-Disposition` subpart header MUST contain a `filename` parameter.
```

Comment:
why was this changed?

Reply:
`embed` didn't seem to be documented in the table bellow., which is why I removed it.